MapboxCoreNavigation/FixLocation.swift

```diff
@@ -9,7 +9,12 @@ extension FixLocation {
             bearingAccuracy = location.courseAccuracy >= 0 ? location.courseAccuracy as NSNumber : nil
         }
         
+        // In practice, “submillisecond precision” is 10 nanosecond precision at best, but convert the timestamp to nanoseconds anyways.
+        // Unlike on Android, we aren’t concerned about the timestamps’ monotonicity.
+        let timestamp = location.timestamp.nanosecondsSince1970
+        
         self.init(coordinate: location.coordinate,
+                  monotonicTimestampNanoseconds: UInt64(timestamp),
                   time: location.timestamp,
```

Comment:
There’s some debate as to whether it’s better for the navigator to use the timestamp of a location update (as we’re currently passing into both `time` and `monotonicTimestampNanoseconds`) or a monotonic system counter (typically used in performance measurements). But it sounds like it’s preferable to leave `monotonicTimestampNanoseconds` unspecified (set to 0) than to fill it in with a location timestamp. At least that way the navigator can make assumptions about the nature of that parameter. The navigator can already fall back to using `time` as before.

/cc @SiarheiFedartsou @kmadsen